pmm-app/src/pmm-dbaas/components/DBCluster/PSMDB.service.ts

```diff
@@ -69,7 +69,7 @@ export class PSMDBService extends DBClusterService {
 
   getDBCluster(dbCluster: DBCluster): Promise<void | DBClusterConnectionAPI> {
```

Comment:
Just minor thing, shouldn't this function be named `getDBClusterCredentials`?

Reply:
Yes it should thanks for pointing it. Will do it in other PR since I started the migration of this code to another project. This can be merged anyway!